### PR TITLE
Add CEX addresses for EVM chains, Bitcoin, Solana, Aptos, Cardano, Algorand, EOS, Cosmos, Elrond, Polkadot, Ripple, Tron, Zilliqa & Litecoin

### PR DESCRIPTION
**Pull request by hildobby**

### CODE REVIEW

File: models/cex/algorand/cex_algorand_addresses.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{{config(
+        tags = ['static'],
+        alias = 'addresses',
```
Comment (jeff-dude) [inline models/cex/algorand/cex_algorand_addresses.sql:3]: can you add `schema` property to all of these new ones?

File: models/cex/algorand/cex_algorand_addresses.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        post_hook='{{ expose_spells(\'["algorand"]\',
```
Comment (jeff-dude) [inline models/cex/algorand/cex_algorand_addresses.sql:4]: do we want all of them exposed on the data explorer?
just double checking. should be fine if so.

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/cex/algorand/cex_algorand_addresses.sql (hunk 1 above)
Comment (hildobby) [reply models/cex/algorand/cex_algorand_addresses.sql:4]: done ☑️

File: models/cex/algorand/cex_algorand_addresses.sql (hunk 2 above)
Comment (hildobby) [reply models/cex/algorand/cex_algorand_addresses.sql:5]: Yes, having all of them exposed is perfect imo

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**